app/javascript/components/enrollment/steps/Review.js

```diff
@@ -71,6 +72,7 @@ class Review extends React.Component {
               details={{ ...this.props.currentState.patient, blocked_sms: this.props.currentState.blocked_sms } || {}}
               authenticity_token={this.props.authenticity_token}
               workflow={this.props.workflow}
+              priority={1}
```

Comment:
This should be passing in `2`, since that's top level header level for the `Patient` component.

Reply:
corrected.